extensions/CubesterYT/ScratchHook.js

```diff
@@ -0,0 +1,168 @@
+(function (Scratch) {
+  "use strict";
+
+  const icon = "https://raw.githubusercontent.com/CubesterYT/dependencies/main/TurboWarp%20Extensions/ScratchHook/Logo.png";
+
+  class ScratchHook {
+    getInfo() {
+      return {
+        id: "cubesterScratchHook",
+        name: "ScratchHook",
+        color1: "#3843B0",
+        menuIconURI: icon,
+
+        blocks: [
+          {
+            opcode: "webhook",
+            text: "webhook data: [hookDATA] webhook url: [hookURL]",
+            blockType: Scratch.BlockType.COMMAND,
+            arguments: {
+              hookURL: {
+                type: Scratch.ArgumentType.STRING
+              }
+            }
+          },
+          {
+            opcode: "params",
+            blockType: Scratch.BlockType.REPORTER,
+            text: "[MENU] [DATA]",
+            arguments: {
+              MENU: {
+                type: Scratch.ArgumentType.STRING,
+                menu: "PARAMS"
+              },
+              DATA: {
+                type: Scratch.ArgumentType.STRING
+              }
+            }
+          },
+          {
+            opcode: "connector",
+            blockType: Scratch.BlockType.REPORTER,
+            text: "[STRING1] , [STRING2]"
+          }
+        ],
+        menus: {
+          PARAMS: {
+            acceptReporters: true,
+            items: ["content","name","icon"]
+          }
+        }
+      };
+    }
+
+    webhook ({hookDATA,hookURL}) {
+      const arrayDATA = hookDATA.split("{,}");
```

Comment:
something tells me it should be possible to shorten a lot of this down

Reply:
Since this is my first time dealing with Extensions, I can't figure out anything else, but this is fully functioning.

Reply:
Due to the fact that it allows for users to not be restricted to a certain order when making webhook messages